developers/ide/eclipse.md

```diff
@@ -14,40 +14,61 @@ For now follow these steps:
+    Eclipse version to use is "Latest Release (2019-03)" (currently version `2019-06` does not support bndtools)
 
-   ![demo import](images/ide_setup_eclipse_import_demo.png)
+1. Under `Github Projects > openHAB` select `openHAB Development` and `openHAB Add-ons`
+    
+   ![select projects](./images/ide_setup_eclipse_projects.png)
 
-   Leave Eclipse open and wait for the downloads and build to finish (about 5-10 minutes).
-1. Using git from the command line, checkout the `openhab2-addons` repository: `git clone --depth=1 https://github.com/openhab/openhab2-addons`
-1. Add the binding that you want to develop (in this guide we use `astro` as an example) to the Eclipse workspace: File -> Import -> Maven -> Existing maven projects -> select the root directory of the binding -> Finish.
+1. Click `Next>`, verify/modify Root and install folder name, click `Next>` and `Finish` to start installation. 
 
-   Leave Eclipse open and wait for the downloads and build to finish.
+    During install accept licence agreement, "Unsigned Content" for Bndtools, and Eclipse Foundation certificates when requested to complete IDE installation. 
 
-   **WARNING**: if the build continues indefinitely, this a bug in the setup that is already being addressed. Meanwhile, to stop the loop deselect from Eclipse -> Project ->  Build Automatically. Remember to do Project -> Build Project after changes in the binding project
+    When the Eclipse installation is finished the IDE is launched automatically.
 
-   *NOTE*: You do not need to add all projects to the workspace, just the binding project you want to work on.
-1. In Eclipse Package Explorer: look for `app.bndrun` in the `org.openhab.demo.app` project.
-   Double click to open `app.bndrun` (takes a few seconds)
-1. Under Browse Repos search for your binding (`astro`) and add it to the "Run Requirements" list using drag&drop from the "Browse Repos" list:
-    ![Bndtools](images/ide_setup_eclipse_bndtools.png)
-1. Click "Resolve": a window with the list of resolved bundles will be shown.
-    Click Finish and save the file.
+1. **It's important**, during the first Eclipse IDE launch, to leave Eclipse open until all openaHAB related initial Startup tasks / Download sources / Builds are completed. 
 
-    Now the IDE is ready to start openHAB with a minimum set of the openHAB core bindings, PaperUI and the selected binding
-1. Start openHAB from the IDE clicking "Run OSGi" or "Debug OSGi" (upper right of the `app.bndrun` dialog).
-   You can check openHAB is running going with your browser to: http://localhost:8080/paperui/
-1. Check the chosen binding is active in PaperUI -> Configuration -> Bindings
+    These tasks will personalize the IDE with openHAB code formatting tools, configurations and a demo app. Initial tasks will also download openHAB lastest SNAPSHOT, the  `openhab2-addons` (bundles) code and related maven dependencies.
+(click bottom right button in the IDE for Progress).
```

Comment:
This looks weird with the brackets and the fullstop behind it.
I am no natove speaker, so does anyone now if this is correct grammatically?

Reply:
TMK, this is not grammatically correct. Even if it was, it just looks ugly :smile:. The 'initial tasks' referred to are 'setup tasks' in terms of the Eclipse Oomph setup, which perform a git clone of the repositories. There is no 'download of the latest snapshot', which is a packaged distribution and has nothing to do with the IDE.

Reply:
done